src/PrestaShopBundle/DependencyInjection/Compiler/LoadServicesFromModulesPass.php

```diff
@@ -37,6 +37,8 @@
 
 /**
  * Load services stored in installed modules.
+ *
+ * @deprecated since 9.0, to be removed in 10.0
```

Comment:
Optmized with the code inside the kernel code, so there's only one place to load modules config files